src/xo-app/backup/overview/index.js

```diff
@@ -351,32 +349,41 @@ export default class Overview extends Component {
                   </tr>
                 </thead>
                 <tbody>
-                  {map(schedules, (schedule, key) => (
-                    <tr key={key}>
-                      <td>{this._getScheduleLabel(schedule)}</td>
-                      <td><a>{this._getScheduleTag(schedule)}</a></td>
-                      <td className='hidden-xs-down'>{schedule.cron}</td>
-                      <td>
-                        {this._getScheduleToggle(schedule)}
-                        <fieldset className='pull-xs-right'>
-                          <ButtonGroup>
-                            <ActionRowButton
-                              icon='delete'
-                              btnStyle='danger'
-                              handler={deleteSchedule}
-                              handlerParam={schedule}
-                            />
-                            <ActionRowButton
-                              icon='run-schedule'
-                              btnStyle='warning'
-                              handler={runJob}
-                              handlerParam={schedule.job}
-                            />
-                          </ButtonGroup>
-                        </fieldset>
-                      </td>
-                    </tr>
-                  ))}
+                  {map(schedules, (schedule, key) => {
+                    const job = this._getScheduleJob(schedule)
+
+                    return (
+                      <tr key={key}>
+                        <td>{this._getJobLabel(job)}</td>
+                        <td><a>{this._getScheduleTag(schedule, job)}</a></td>
+                        <td className='hidden-xs-down'>{schedule.cron}</td>
+                        <td>
+                          {this._getScheduleToggle(schedule)}
+                          <fieldset className='pull-xs-right'>
+                            <Link to={`/backup/edit/${job.id}`}>
+                              <button className='btn btn-sm btn-primary m-r-1'>
```

Comment:
Do not put a button inside a link, what you might do is to add these classes directly to the `Link`.